src/Makefile.am

```diff
@@ -156,6 +157,7 @@ BITCOIN_CORE_H = \
   util.h \
   utilmoneystr.h \
   utiltime.h \
+  utilstr.h \
```

Comment:
Should this be `utilstring.h`?

Reply:
Yes! doing small fix pr - apologies.

Reply:
All good, I accidentally used `utilstr.h` instead of `utilstring.h`.